ocs_ci/ocs/resources/mcg.py

```diff
@@ -1016,6 +1016,7 @@ def reset_core_pod(self):
         """
 
         self.core_pod.delete(wait=True)
+        sleep(60)
```

Comment:
is there some resource state we can sample here with an interval of a few seconds?
Otherwise, please add a log message that we are going to wait for 60 seconds

Reply:
Since we're deleting the pod with `wait=True` then after this line we should have 0 noobaa-core pods for a short time period. To make sure the new pod exists at least at "Creating" before attempting to store its data and check its status in the following commands, we can use [wait_for_pods_by_label_count](https://github.com/red-hat-storage/ocs-ci/blob/27a124581690d1adc6080faa2b080bb1f360ac27/ocs_ci/ocs/resources/pod.py#L2403-L2439) to wait until we go from 0 noobaa-core pods to 1:

```
        wait_for_pods_by_label_count(
            label=constants.NOOBAA_CORE_POD_LABEL,
            exptected_count=1,
        )

Reply:
done